### ISSUE TITLE
FSL 6.0.4 not avaible for docker?

### ISSUE THREAD

**Issue by fhopp** [ReproNim/neurodocker#410]
Hi everyone, 

it says that FSL version 6.0.4 is available for install, but running: 

`docker run --rm repronim/neurodocker:0.7.0 generate docker \
    --base debian:stretch --pkg-manager apt --fsl version=6.0.4` 

yields: 
```
# Generated by: Neurodocker version 0.7.0+0.gdc97516.dirty
# Latest release: Neurodocker version 0.7.0
[NEURODOCKER 2021-06-30 00:01:46,742 ERROR]: Failed to instantiate <class 'neurodocker.interfaces.interfaces.FSL'>: URL not found for version '6.0.4'. Valid versions are 6.0.3, 6.0.2, 6.0.1, 6.0.0, 5.0.11, 5.0.10, 5.0.9, 5.0.8
Traceback (most recent call last):
  File "/usr/bin/neurodocker", line 11, in <module>
    load_entry_point('neurodocker', 'console_scripts', 'neurodocker')()
  File "/opt/neurodocker/neurodocker/neurodocker.py", line 429, in main
    subparser_functions[namespace.subsubparser_name](namespace)
  File "/opt/neurodocker/neurodocker/neurodocker.py", line 373, in generate
    print(recipe_obj.render())
  File "/opt/neurodocker/neurodocker/generators/docker.py", line 297, in render
    self._ispecs_to_dockerfile_str())
  File "/opt/neurodocker/neurodocker/generators/docker.py", line 328, in _ispecs_to_dockerfile_str
    interface = impl(pkg_manager=pkg_man, **params)
  File "/opt/neurodocker/neurodocker/interfaces/interfaces.py", line 126, in __init__
    super().__init__(self._name, *args, **kwargs)
  File "/opt/neurodocker/neurodocker/interfaces/_base.py", line 204, in __init__
    self.binaries_url = self._resolver.binaries_url(self._version)
  File "/opt/neurodocker/neurodocker/interfaces/_base.py", line 170, in binaries_url
    self.check_binaries_has_url(version)
  File "/opt/neurodocker/neurodocker/interfaces/_base.py", line 166, in check_binaries_has_url
    .format(version, ', '.join(valid_vers))
ValueError: URL not found for version '6.0.4'. Valid versions are 6.0.3, 6.0.2, 6.0.1, 6.0.0, 5.0.11, 5.0.10, 5.0.9, 5.0.8
``` 

Any idea what could be causing the issue? 
Thanks much in advance!

**Comment by stebo85** [comment]
Dear @fhopp,

Could the problem be that the neurodocker version you have installed is not up-to date with the git repo here?

Would that work:?
pip install --no-cache-dir https://github.com/kaczmarj/neurodocker/tarball/master --user

We build fsl 6.0.4 in our neurodesk project with neurodocker and it works :)
https://github.com/NeuroDesk/caid/packages/444624
https://hub.docker.com/r/vnmd/fsl_6.0.4/tags?page=1&ordering=last_updated

Maybe that helps already :)
Cheers
Steffen

**Comment by kaczmarj** [comment]
@fhopp - thanks for the report. as @stebo85 mentioned, can you please try the latest version of neurodocker?

```
docker run --rm repronim/neurodocker:master generate docker \ 
  --base debian:stretch --pkg-manager apt --fsl version=6.0.4
```

thanks @stebo85

**Comment by fhopp** [comment]
Hey @kaczmarj , 
does this update also fix #406 ? 

I am also doing all this in Ubuntu 20.04 as our workstations all run 20.04 and I need to maintain the same OS in docker or I will get slightly different results due to OS differences in floating point precision etc.

**Comment by kaczmarj** [comment]
> does this update also fix #406 ?

Not yet, but i have opened a pull request at #411 to fix it
